SitecoreDeployer/buildtask/functions-module-v1.psm1

```diff
@@ -258,7 +258,7 @@ function SetWebAppRestrictions {
     }
 
     #get all resrouces in current resource group, and check if resource is present there
-    $webAppResource = (Find-AzureRmResource -ResourceGroupNameContains $resourceGroupName).where({$_.Name -eq "$webAppInstanceName" -And $_.ResourceGroupName -eq "$ResourceGroupName"})
+    $webAppResource = (Get-AzureRmResource -ResourceGroupName *$resourceGroupName*).where({$_.Name -eq "$webAppInstanceName" -And $_.ResourceGroupName -eq "$ResourceGroupName"})

```

Comment:
Please, remove asteriks here - we need resources from specific resource group, not matching one (in Find-AzureRmResource there was not possibility to match exact rg name)